### ISSUE TITLE
Create a new logrotate.Option "MaximumFiles"

### ISSUE THREAD

**Issue by xinzhli** [easyCZ/logrotate#2]
There is currently no setting for deleting files. If there are too many files, is it better to set a parameter “MaximumFiles” to delete them?

**Comment by easyCZ** [comment]
Good idea @xinzhli, I'm happy to accept contributions if you'd like to contribute.

Otherwise, I can look into adding it.
